opensearch/services/datasets/datasets.php

```diff
@@ -358,6 +395,7 @@ public function getQueryBody($type) {
 				$body .= "BIND(str(?g_l) AS ?gp_label) . ";
 				$body .= "BIND(str(?acc) AS ?access) . ";
 				$body .= "BIND(str(?pl) AS ?project_label) . ";
+				$body .= "BIND(str(?d_l) AS ?dataType_label) . ";
```

Comment:
This BIND statement should probably be in the OPTIONAL clause associated with the ?d_l assignment (line 390)